README.md

```
@@ -1,171 +1,47 @@
-# Over the Board 🏆
+# Chess Tournaments Database Archive
 
-Bem-vindo à **Over the Board**! Uma API moderna e robusta para consulta de informações sobre xadrez da Confederação Brasileira de Xadrez (CBX).
+Chess tournament data with 387995 FIDE tournaments.
 
-> **Nota**: Este é um projeto pessoal e não possui ligação oficial com a CBX.
+**This project is discontinued**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)